test/e2e/configure_cni_test.go

```diff
@@ -73,9 +73,12 @@ func testConfigureCNI(t *testing.T) {
 	time.Sleep(10 * time.Second)
 
 	assert.True(t, isKubeletRunning(t, kubeletLogPath))
-	isConfiguredCorrectly, err := isCNIConfigured(t, kubeletLogPath)
-	require.NoError(t, err, "Error reading kubelet log")
-	assert.True(t, isConfiguredCorrectly, "CNI was not configured correctly")
+	// if the runtime is containerd, the network plugin is loaded at a later stage.
+	if dockerRuntime {
+		isConfiguredCorrectly, err := isCNIConfigured(t, kubeletLogPath)
+		require.NoError(t, err, "Error reading kubelet log")
+		assert.True(t, isConfiguredCorrectly, "CNI was not configured correctly")
+	}
```

Comment:
Not sure if I understand correctly -- if the node picks up CNI networking changes later with containerd, would we be able to run this test later on in the test suite? Or wait for some time and then run it? I just don't want to skip this test case entirely if containerd is the runtime.

Reply:
We are suspecting this log might be actually coming from docker runtime hence we are not seeing it in containerd case, I am looking into it more closely now will update soon.